project/Dependencies.scala

```diff
@@ -183,6 +184,7 @@ object Dependencies {
   val backendDeps = Def.setting(Seq(
     "com.typesafe.scala-logging" %% "scala-logging" % scalaLoggingVersion,
     "ch.qos.logback" % "logback-classic" % logbackVersion,
+    "org.codehaus.janino" % "janino" % janinoVersion,
```

Comment:
Can't really tell where it is used, could you clarify?

Reply:
Welcome to the Java world, where library capabilities are determined on runtime based on the JARs that are loaded. Basically conditional statements in logback require this - adding a comment.